docs/IntegrationWithExistingApps.md

```diff
@@ -584,13 +588,15 @@ allprojects {
         ...
         maven {
             // All of React Native (JS, Android binaries) is installed from npm
-            url "$rootDir/node_modules/react-native/android"
+            url "$rootDir/../node_modules/react-native/android"
         }
     }
     ...
 }
 ```
 
+Make sure that the path is correct! You shouldn’t run into any “Failed to resolve: com.facebook.react:react-native:0.x.x" errors after running Gradle sync in Android Studio.
```

Comment:
You could make this a note via prepending a ">" in front of the paragraph. Maybe a note is better here.